x-pack/plugins/security_solution/public/common/containers/hosts_risk/use_hosts_risk_score.ts

```diff
@@ -40,9 +39,17 @@ export interface HostRisk {
 export const useHostsRiskScore = ({
   timerange,
   hostName,
+  onlyLatest = true,
+  queryId = QUERY_ID,
+  sortOrder,
+  limit,
 }: {
   timerange?: { to: string; from: string };
   hostName?: string;
+  onlyLatest?: boolean;
+  queryId?: string;
```

Comment:
Could we put queryId as an enum, so it gathers all the queryIds for hostRiskScore?